docs/trino-connector/configuration.md

```diff
@@ -6,8 +6,8 @@ license: "Copyright 2023 Datastrato Pvt Ltd.
-| gravitino.url      | string | http://localhost:8090   | The `gravitino.url` defines the connection URL for the Gravitino server. If not set, the default value is `http://localhost:8090`. If the Gravitino server is not starting, Trino can still start normally. Once the Gravitino server is up and running, the Gravitino connector will automatically connect. You can find error details in the Trino logs.                            | Yes      | 0.2.0         |
+| Property           | Type   | Default Value         | Description                                                                                                                                                                                                                                                                                                                                                                                                                         | Required | Since Version |
+|--------------------|--------|-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------|---------------|
+| connector.name     | string | (none)                | The `connector.name` defines the name of Trino connector, this value is always 'gravitino'.                                                                                                                                                                                                                                                                                                                                         | Yes      | 0.2.0         |
+| gravitino.metalake | string | (none)                | The `gravitino.metalake` defines which metalake in Gravitino server the Trino connector uses. Trino connector should set it at start, the value of `gravitino.metalake` need to a valid name of metalake, no matter Gravitino has created it or not, once Trino connector successfully detect the existence of metalake, it will load all catalogs, schemas, and tables under this metalake automatically and keep synchronization. | Yes      | 0.2.0         |
+| gravitino.url      | string | http://localhost:8090 | The `gravitino.url` defines the connection URL of the Gravitino server, the default value is `http://localhost:8090`. If this Gravitino server has not started, Trino can still start and work normally. Once the Gravitino server is up and running, Trino connector will automatically connect to Gravitino sever and load metadata.                                                                                              | Yes      | 0.2.0         |
```

Comment:
Trino connector can detect and connect to Gravitino server once it is ready, no need start Gravitino beforehand.